src/Shell/Task/ModelTask.php

```diff
@@ -337,7 +340,7 @@ public function findHasMany($model, array $associations)
                         'alias' => $otherModel->alias(),
                         'foreignKey' => $fieldName
                     ];
-                } elseif ($otherTable === $tableName && $fieldName === 'parent_id') {
+                } elseif ($otherTableName === $tableName && $fieldName === 'parent_id') {
```

Comment:
*Revisiting this unfinished PR because I got issues with composite foreign keys (and I want to make sure composite primary keys work, first).*

This code ABOVE to ModelTask above somehow made it into master (not via my PR).